### PR TITLE
Added paragraph about feedback acceptance

### PR DESCRIPTION
**Pull request by kirscfil**

### CODE REVIEW

File: en/applifters-codex.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ If you have a conflict with another Applifter, do not hold it inside. First, try
 
  Only with honesty and sincerity will we be able to keep Applifting a place where it is a joy to be working and where we can talk openly.
 
+### I can accept feedback
```
Comment (petrbazout) [inline en/applifters-codex.md:31]: How are you going to bring people higher in EQ if they are not on a level where they can accept feedback?

Comment (kirscfil) [reply en/applifters-codex.md:31]: I truly do not know if it's possible. I think that the ability to accept feedback is something which is implicitly part of our culture - and this step only makes it explicit.

Comment (petrbazout) [reply en/applifters-codex.md:31]: @kirscfil So you have to hire people who can already accept feedback, right? How do you select for it in the interview process?

File: cs/applifters-codex.md [hunk 2]
```diff
@@ -118,3 +122,4 @@ Jsem hrdý na to, co umím. Rád sdílím to, co jsem se naučil s ostatními. A
 | v0.4  | 31.8.2018  | Add basic assumptions                               | Vráťa |
 | v0.5  | 12.12.2018 | Add paragraph about interpersonal conflict handling | Vráťa |
 | v0.6  | 25.3.2019  | Add conflict resolution and non-toxicity chapter    | Vráťa |
+| v0.7  | 8.10.2019  | Add paragraph about feedback acceptance             | Filip |
```
Comment (ViliamV) [inline cs/applifters-codex.md:125]: typo :slightly_smiling_face:  
```suggestion
| v0.7  | 8.10.2020  | Add paragraph about feedback acceptance             | Filip |
```

File: en/applifters-codex.md [hunk 3]
```diff
@@ -112,6 +116,7 @@ I am proud of my skills. I like to share my expertise with others, be it in the
  
 | Version | Date       | Change                          | Author | Corresponding CZ version |
 | ------- | ---------- | ------------------------------- | ------ | ------------------------ |
+| v0.8    | 2019/10/08 | Translate feedback acceptance   | Filip  | v0.7                     |
```
Comment (ViliamV) [inline en/applifters-codex.md:119]: same
```suggestion
| v0.8    | 2020/10/08 | Translate feedback acceptance   | Filip  | v0.7                     |
```

File: en/applifters-codex.md (hunk 1 above)
Comment (kirscfil) [reply en/applifters-codex.md:31]: @petrbazout Eiter me or @Vratislav are present on each interview and assure the cultural compatibility (especially talking about conflicts, feedback and asking about past experience with these).

**Comment by martinsrb** [comment]
I suggest to remove the word constructive from the proposed change. I agree that feedback shall be  constructive however the issue is who judge that. The giving side can consider it as constructive where the receiving side can see it otherwise. Sometimes it is the form rather the content which causing the twist of perception. I believe that openness requites being able to receive feedback without adjectives to prevent self-censorship.

**Pull request merged**